src/Collector/ExcludeInternalCollector.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Qossmic\Deptrac\Collector;
+
+use Qossmic\Deptrac\AstRunner\AstMap;
+use Qossmic\Deptrac\AstRunner\AstMap\AstClassReference;
+
+class ExcludeInternalCollector implements CollectorInterface
+{
+    public function getType(): string
+    {
+        return 'internal';
```

Comment:
I would like a more expressive name like `marked_internal` or `internal_classes` instead. Any suggestions?

Reply:
I would prefer `marked_internal` as `internal_classes` could be interpreted as a php internal class